packages/fluentui/docs/src/examples/components/Popup/Variations/PopupExampleOffset.shorthand.tsx

```diff
@@ -3,7 +3,13 @@ import { Button, Grid, Popup } from '@fluentui/react-northstar';
 import { ArrowUpIcon } from '@fluentui/react-icons-northstar';
 
 const PopupExampleOffset = () => (
-  <Grid columns="1, 80px" variables={{ padding: '30px' }}>
+  <Grid
+    styles={{
+      gridTemplateColumns: '1, 80px',
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/81697385-4779e500-9465-11ea-8275-628f429e60fb.png)

It does not look like a valid style

Reply:
Yeah, I am not sure what was the intention here. any value I put there `1fr` for example will generate regression in the screener... So what is supposed to be the correct value? @layershifter @pompomon

Reply:
Let's fix it with a valid meaningful style even if there will be a regression.

Reply:
Alright, I am setting as `1fr`, we can decide if needs to be changed

Reply:
Having a look in the screener results now, what would be the proper value here? @layershifter @pompomon